task1/task1_test.go

```diff
@@ -0,0 +1,13 @@
+package task1
+
+import (
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"testing"
+)
+
+func TestServer(t *testing.T) {
+	response, err := http.Get("http://127.0.0.1:5050/anything/you?want=learn")
+	assert.Nil(t, err, response)
```

Comment:
`assert.NoError` can be used.
Speaking of testing, you'd want to read response and parse it. Also, it's seems to be like not just a unit test and package can be named as `task1_test`.
Anyway, it's good to write some tests.